demo/App.vue

```diff
@@ -10,8 +10,9 @@ v-app.app
             :close-on-content-click="false",
             content-class="girder-search-arrow-menu",
             v-model="uiOptionsMenu")
-          v-btn(icon, slot="activator")
-            v-icon.mdi-24px {{ $vuetify.icons.more }}
+          template(#activator="{ on }")
+            v-btn(icon, v-on="on")
+              v-icon.mdi-24px {{ $vuetify.icons.values.more }}
```

Comment:
I would like to note one thing here. v-icon could take strings directly, it was a side effect that it worked with or without the {{ }}. In Vuetify 2, the designed usage is v-icon with string `$vuetify.icons.more`, but there is a lookup logic happening on the string which seems a little bit unnecessary to me personally.